website/docs/tutorials/quickstart.md

```diff
@@ -306,7 +312,7 @@ unleash.on('synchronized', () => {
   // Unleash is ready to serve updated feature toggles.
 
   // Check a feature flag
-  const isEnabled = unleash.isEnabled('some-toggle');
+  const isEnabled = unleash.isEnabled('some-flag');
 
   // Check the variant
   const variant = unleash.getVariant('app.ToggleY');
```

Comment:
We should probably update this one too, right? I don't know whether we want to use the same flag (as in my suggestion) or we'd rather stick to using a different flag name (such as `some-other-flag`). What do you think?

```suggestion
  const variant = unleash.getVariant('some-flag');
```

Reply:
Toggle isn't a forbidden word. I left this as an example where the mental model of a toggle (binary, on/off) might be better than a flag. So I left it as-is.